### ISSUE TITLE
Getting an exception when running gauge specs

### ISSUE THREAD

**Issue by sguptatw** [getgauge/gauge#452]
### Expected behavior
I should be able to run specs from command line

### Actual behavior
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x20 pc=0x401c54]

goroutine 1 [running]:
panic(0xa77040, 0xc082008070)
        /usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:464 +0x3f4
main.main()
        /Users/stechops/Library/Application Support/Go Agent/pipelines/Gauge-Nightly/gauge.go:152 +0xc14
```

### Steps to reproduce
1. Run gauge specs in the test project
[test.zip](https://github.com/getgauge/gauge/files/306801/test.zip)

### Gauge version
```
Gauge version: 0.4.1.nightly-2016-06-08

Plugins
-------
csharp (0.8.0.nightly-2016-06-08)
html-report (2.1.1.nightly-2016-06-02)
java (0.4.1.nightly-2016-05-30)
ruby (0.2.0)
xml-report (0.1.2.nightly-2016-05-03)
```

**Comment by karolxavierleite** [comment]
I got the same error. Using ruby and running on MacOs.

## Gauge version

```
Gauge version: 0.5.0

Plugins
-------
html-report (2.1.0)
ruby (0.2.1)
spectacle (0.0.1)
```

**Comment by Tpbrown** [comment]
@karolxavierleite @sguptatw  I had this problem too, on both Ubuntu and Alpine.

Appears to be fixed in 0.5.1 nightly build - https://bintray.com/gauge/Gauge/Nightly/

**Comment by kashishm** [comment]
This issue is fixed in the latest release 0.6.0. @sguptatw please close this issue after verifying.
